requirements.txt

```diff
@@ -1,4 +1,4 @@
-qiskit>=0.4.0
+qiskit>=0.46.0
```

Comment:
should we leave this out so that the user can select the version? I've seen issues with qiskti 1.0 not being downgrade-able so I'm a little hesitant to have a default dependency on it.

Reply:
we wanted to add a dependencies for `>=0.46.0` and `>=1.0.0` versions but couldnt find a good way to do that with pip. maybe just having `qiskit` be a dependency without a version until `0.46.x` is unsupported would be fine?

Reply:
> I ran into this comment while searching for something else. With which version of qiskit is qiskit-ionq compatible with? I don't fully get the advantage of removing the dependency and failing at import time.

Qiskit 1.0 was not compatible with many qiskit ecosystem libs at launch. If we were to leave qiskit without a version in the requirements, then it would by default install qiskit at the latest version, which may be 1.0+, and cause issues for users who want to use pre-qiskit-1.0 libraries.

By removing the dependency entirely and failing on import with a warning, we allow the user to select whether they would like to use qiskit 1.0 or qiskit < 1.0 without requiring them to 'undo' the changes that the install made for them.

Theoretically qiskit without a version dependency would be fine, @splch , but I was worried that it was no longer possible to `pip install qiskit-ionq` and get the same behavior re: support for other libraries from scratch, because 'from scratch' you now default get qiskit 1.0

Anyways, I would be fine with qiskit without a particular version, but I'd like to have docs about how to downgrade from qiskit 1.0 to 0.46 in that case, since it seems like a path that users may have to deal with.

Reply:
If you install qiskit 0.45 and qiskit 1.0 into the same python environment, they will fail, because qiskit 0.45 install qiksit-terra 0.45 and this later conflicts with qiskit 1.0. This seems like a fundamentally flawed upgrade path and I am leery to put people on it by default, I suppose.